modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZTextWithMediaParagraphBehavior.php

```diff
@@ -248,6 +248,13 @@ public function preprocess(&$variables) {
     if (!empty($config['bg_color']) && $config['bg_color'] !== 'dark') {
       $title_classes[] = 'text-blue';
     }
+    if ((!empty($config['style']) && $config['style'] !== 'bottom') && (!empty($config['bg_color']) && $config['bg_color'] !== 'dark')) {
+      $content_classes = [
+        'bg-white',
+        'shadow',
+        'mb-4',
+      ];
+    }
```

Comment:
Bad formatting, but that should do what you want, as long as you place this block above line 241

Reply:
Removed !empty checks because we have default settings. 
https://github.com/az-digital/az_quickstart/pull/1683/files#diff-3835e62d0ade8930c6536b446dfe773d6022f3eee035e46dbee2b12ebcbfb8afR39-R48